tests/custom_strategies/__init__.py

```diff
@@ -313,3 +314,53 @@ def adv_integer_index(
             min_dims=min_dims, max_dims=max_dims, min_side=min_side, max_side=max_side
         ),
     )
+
+
+@lru_cache(maxsize=1000)
+def _factors(n: int) -> List[int]:
+    """ Returns the divisors of n
+
+        >>> _factors(4)
+        {1, 2, 4}"""
```

Comment:
and here

Reply:
I'll do a sweep through the whole library and take care of it all at once